packages/augur-ui/src/modules/auth/actions/update-assets.ts

```diff
@@ -45,6 +53,12 @@ function updateBalances(
     const legacyRep = formatAttoRep(legacyAttoRep).value;
     const dai = amounts[1];
     const eth = amounts[2];
+    if (
```

Comment:
move this logic out of update-assets. update-assets shouldn't care about analytics